### PR TITLE
Optimize did:web routes

### PR DESCRIPTION
**Pull request by TaylorBeeston**
# Overview

#### 📚 What is the context and goal of this PR?
LearnCloud's main did:web route has been randomly throwing 500s, and it looks like it's because the
connection to Mongo is failing (something that doesn't even need to happen for did:web!)

#### 🥴 TL; RL:
Packages each lambda function separately and moves the did web handler into its own file, meaning that
all the mongo code doesn't even get packaed with that lambda!

#### 🛠 Important tradeoffs made:
I did the same change for LCN, then immediately realized that we actually have to connect to Neo4j
for the did:web route in order to create each user's did doc =/ thankfully we aren't seeing the same
error there (yet)

#### 🔍 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Chore (refactor, documentation update, etc)

#### 💳 Does This Create Any New Technical Debt? ( If yes, please describe and [add JIRA TODOs](https://welibrary.atlassian.net/jira/software/projects/WE/boards/2) )
- [x] No
- [ ] Yes

# Testing

#### 🔬 How Can Someone QA This?

I'm not sure! I checked it by deploying to my local AWS and verifying that did:web still worked...

#### 🧪 Code Coverage

# Documentation

#### 📊 Storybook

# ✅ PR Checklist
- [ ] Related to a Jira issue ([create one if not](https://welibrary.atlassian.net/jira/software/projects/WE/boards/2))
- [x] My code follows **style guidelines** (eslint / prettier)
- [x] I have **manually tested** common end-2-end cases
- [x] I have **reviewed** my code
- [x] I have **commented** my code, particularly where ambiguous
- [x] New and existing **unit tests pass** locally with my changes
- [ ] I have made corresponding changes to **gitbook documentation**

### 🚀 Ready to squash-and-merge?:
- [x] Code is backwards compatible
- [x] There is **not** a "Do Not Merge" label on this PR
- [x] I have thoughtfully considered the security implications of this change.
- [x] This change does not expose new public facing endpoints that do not have authentication

### CODE REVIEW

File: services/learn-card-network/learn-cloud-service/serverless.yml [hunk 1]
```diff
@@ -9,8 +9,8 @@ useDotenv: true
 plugins:
     - serverless-esbuild
     - serverless-offline
-    - serverless-domain-manager
-    - serverless-certificate-creator
+    # - serverless-domain-manager
+    # - serverless-certificate-creator
```
Comment (Custard7) [inline services/learn-card-network/learn-cloud-service/serverless.yml:14]: Do we not need these anymore?

File: services/learn-card-network/learn-cloud-service/serverless.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ custom:
         packager: pnpm
         plugins: esbuildPlugins.cjs
         bundle: true
-        minify: true
```
Comment (Custard7) [inline services/learn-card-network/learn-cloud-service/serverless.yml:22]: Same with this?

**Review by Custard7 — APPROVED** [review]
Some questions about the commented out lines, but otherwise looks good.

Let's monitor closely during deploy 👍 

And verify that all plugfest / chapi routes are working well still since they will be experiencing traffic today/rest of this week.

**Comment by TaylorBeeston** [comment]
@Custard7 good catch! Those were for my testing/debugging! 😂

**Pull request merged**